### DISCUSSION TITLE
Dependency between alerts

### DISCUSSION THREAD

**Discussion by Nayruden** [jertel/elastalert2#1521] [Q&A] [answered]
Is there any way to create a dependency or information flow between alerts? My specific use case is that I create a GitLab issue using `post2` and a Mattermost message using `mattermost`. I'd like to have the Mattermost message reference the issue that was created on GitLab, and that information is available in the returned data from the HTTP POST.

**Comment by jertel** [comment, marked as answer]
No, it's not currently possible with the base project. You could accomplish it by creating a custom Python alerter that wrapped/extended the http_post2 alerter. It would take the response, unpack the GitLab issue ID, and then set that into a match field, or alerter parameter for the Mattermost alerter.

**Reply by Nayruden** [reply]
I figured it would be something like that. Thanks!
